draft-ietf-mls-protocol.md

```diff
@@ -1750,11 +1866,45 @@ introduced.  The only choices the creator makes here are its own
 KeyPackage, the leaf secret from which the Commit is built, and the
 intermediate key pairs along the direct path to the root.
 
+## Group Creation from existing PSKs
+
+Group creation may be tied to an already existing group structure, consisting of
+re-initialization of an existing group, or branching of a sub-group.
+
+Re-initialization of an existing group may be used, for example, to re-start the
+group based on the current group state but under a different ciphersuite.
+
+Branching may be used to bootstrap a new group consisting of a subset of
+current group members, based on the current group state.
+
+In both cases, the `psk_nonce` included in the `PreSharedKeyID` object must be a
+randomly sampled nonce to avoid key re-use.
```

Comment:
I've specified the nonce length to be `KDF.Nh` and added an open issue regarding the possibility to avoid the nonce altogether using the GroupId of the new group.